src/components/PetCard.tsx

```diff
@@ -65,7 +67,7 @@ export const PetCard = ({
               )}
               <CardDescription className="flex items-center gap-1">
                 <TbClockHour3 />
-                {dateOfBirth}
+                <DateOfBirth dateOfBirth={date_of_birth} />
```

Comment:
I would revert to how it was before since it's basically one line of code, so that it's easier to read